TeamShiksha/logoexecutive
# Added Business API to get free logos

### Issue
Feature - New API - /images/logo [#120 ](https://github.com/TeamShiksha/logoexecutive-backend/issues/120)

### Description
This PR introduces a new GET api endpoint at /api/images/logo that allows the user to fetch the logo of a particular company.
When the user fetches a logo by entering a company name, this API is called, which checks the firestore for the company logo, and if present returns the Cloudfront CDN link. The user can then go to this link, and download the logo

### How to Test the Changes
Start the server locally.
Call the signin API first to sign in. API requires the user to be signed in.
Send a GET request to /api/images/logo specifying the companyName and apiKey as query parameters.
Verify that the endpoint responds with a status code of 200 and a success message with a CDN link to the Image. Go to this CDN link and confirm if the image is present.



https://github.com/TeamShiksha/logoexecutive/assets/46561984/3f0681de-eb8c-4ce5-8cfe-23d22f85cf7f
